ee/tables/sleeper/table.go

```diff
@@ -0,0 +1,62 @@
+package sleeper
+
+import (
+	"context"
+	"log/slog"
+	"strconv"
+	"time"
+
+	"github.com/kolide/launcher/ee/agent/types"
+	"github.com/kolide/launcher/ee/tables/tablehelpers"
+	"github.com/kolide/launcher/ee/tables/tablewrapper"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+type Table struct {
+	slogger *slog.Logger
+}
+
+func TablePlugin(flags types.Flags, slogger *slog.Logger) *table.Plugin {
+
+	columns := []table.ColumnDefinition{
+		table.IntegerColumn("duration"),
+	}
+
+	t := &Table{
+		slogger: slogger.With("table", "kolide_deadly_sleeper"),
+	}
+
+	return tablewrapper.New(flags, slogger, "kolide_deadly_sleeper", columns, t.generate, tablewrapper.WithTableGenerateTimeout(10*time.Minute))
+
+}
+
+func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	var results []map[string]string
+
+	for _, durationStr := range tablehelpers.GetConstraints(queryContext, "duration") {
+		if err := func() error {
+			duration, err := strconv.Atoi(durationStr)
+			if err != nil {
+				return err
+			}
+
+			t.slogger.Log(ctx, slog.LevelWarn, "The deadly sleeper table sleeps!", "duration", duration)
+
+			ticker := time.NewTicker(time.Duration(duration) * time.Second)
+			defer ticker.Stop() // nolint:revive // we want to
```

Comment:
For additional clarity, maybe --

```suggestion
			defer ticker.Stop() // nolint:revive // we don't expect a large number of duration strings in the query, so we think it is fine to accumulate defers inside the for loop here to keep the code simpler
```

...or something like that, but less wordy 😅

Reply:
Oh. Ha. I didn't mean to commit that. There's a anonymous function to make the linter happy 😆

Reply:
Oh yup, I see it now! Might be worth a comment at the top of the anonymous func to explain that we use it to avoid stacking up defers, then